team-validator.js

```diff
@@ -22,7 +22,11 @@ class Validator {
 		if (supplementaryBanlist && supplementaryBanlist.length) {
 			format = Object.assign({}, format);
 			if (format.banlistTable) delete format.banlistTable;
-			if (!format.banlist) format.banlist = [];
+			if (format.banlist) {
+				format.banlist = format.banlist.slice(0);
```

Comment:
To make a copy of an array, it's just `.slice()`.

It does the same thing as `.slice(0)` but I think it's clearer.